Frontend/android-native/mobility-customer/src/main/java/in/juspay/mobility/customer/MobilityCustomerBridge.java

```diff
@@ -134,6 +139,33 @@ public void inAppCallBack(String onTapAction) {
         }
     }
 
+    @JavascriptInterface
+    public void saveOfflineDriverInfo(String _Driverinfo){
+        try {
```

Comment:
pass the DB_NAME, TAVLE_NAME etc from purescript make generic functions to read ,write ,create and delete tables and DB.